test/BTCUtilsJs.test.js

```diff
@@ -0,0 +1,442 @@
+/* global artifacts contract before it assert */
+// const BN = require('bn.js');
+const constants = require('./constants');
+
+const assert = require('chai').assert
+
+const HEADER_170 = '0x0100000055bd840a78798ad0da853f68974f3d183e2bd1db6a842c1feecf222a00000000ff104ccb05421ab93e63f8c3ce5c2c2e9dbb37de2764b3a3175c8166562cac7d51b96a49ffff001d283e9e70';
```

Comment:
Deserialize here?  (and for the rest of the `const`s on the next several lines)

Reply:
done